Source/WebCore/platform/graphics/haiku/FontCustomPlatformData.cpp

```diff
@@ -46,6 +46,10 @@ std::unique_ptr<FontCustomPlatformData> createFontCustomPlatformData(SharedBuffe
 	BFont font;
 
 	status_t result = font.LoadFont(area, buffer.size(), 0);
+
+    if (result == B_NOT_SUPPORTED)
```

Comment:
Shouldn't the check be `!= B_OK`? I guess we'll want the same for whatever error we get.